frontend/components/feedback-task/SearchBar.base.vue

```diff
@@ -48,10 +48,14 @@
       @keydown.enter.exact.stop=""
     />
 
-    <span class="search-area__additional-info" v-text="additionalInfo" />
+    <span
+      class="search-area__additional-info"
+      v-if="!searchValueIsEmpty"
```

Comment:
I think we should show the additional info only when the query search prop is the same as the query search in the search bar